neurogym/wrappers/monitor.py

```diff
@@ -461,3 +496,120 @@ def plot_training_history(
             logger.info(f"Figure saved to {save_path}")
 
         return fig
+
+    def record_activations(
+        self,
+        layer: nn.Module,
+        name: str | None = None,
+        steps: int | None = None,
+    ) -> ActivationMonitor:
+        """Record the output activations of a layer over a trial.
+
+        Args:
+            layer: The layer whose activations are being monitored.
+            name: The name to use for the activation monitor.
+                This can be useful for retrieving activation monitors
+                at a later stage.
+            steps: The steps to record for.
+                This could be less than the total number of steps in a trial.
```

Comment:
Looking at `ActivationMonitor`, I think this is specifically the first n steps, right?

```suggestion
            steps: The number of steps to record for. Will record the first steps until this number is reached. Defaults to the total number of steps.
```

Even fancier would be to use a slice, so that users can give a start, end, and interval if they want. Not sure it's worth the effort to go the extra mile for this, though.